src/Adapters/ADTAdapter.ts

```diff
@@ -100,16 +98,12 @@ export default class ADTAdapter implements IADTAdapter {
     constructor(
         adtHostUrl: string,
         authService: IAuthService,
-        tenantId?: string,
```

Comment:
@darsney is this gonna cause a lot of pain for consumers who already provide these today? Should we send a comm to Paul about this so they aren't surprised if they pull latest versions in?

Reply:
Yeah, I moved the tenantId to the authService constructor which makes more sense. Lets discuss the breaking changes on Monday.